doc/ref_model/chapters/chapter07.rst

```diff
@@ -354,8 +368,9 @@ security activities need to determine whether the requirements of the process ch
 of the security activities meets the requirements, the next set of security activities are performed as the process
 transitions to the next checkpoint, or, alternatively, to the next stage if the checkpoint is the last one in the
 current stage. If, on the other hand, the outcome of the security activities does not meet the requirements, then the
-process should not be allowed to advance to the next checkpoint. Tables 7-9 to 7-13 in Section 7.9 define the IaaC
-security activities presented as security requirements mapped to particular stages and trigger points.
+process should not be allowed to advance to the next checkpoint. In the section 
+":ref:`ref_model/chapters/chapter07:consolidated security requirements`", the IaaC
```

Comment:
@pgoyal01 I prefer the ref directive as it would work everywhere in RM. Your proposal would be limited to a single rst file.
Sphinx seconds this point in https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-ref

It's true that today :ref: falsy replaces internal links by external ones because of the CNTT readthedocs builds.
I'm hoping we would agree to change that next Weekly Technical meeting - right now I would recommend to preprocess the :ref: as proposed here https://github.com/cntt-n/CNTT/pull/3013